pkg/dao/repository_test.go

```diff
@@ -38,31 +16,31 @@ func (suite *ReposSuite) TestCreate() {
 	t := suite.T()
 
 	found := models.RepositoryConfiguration{}
-	db.DB.First(&found)
+	suite.tx.First(&found)
 
-	err := GetRepositoryDao().Create(api.RepositoryRequest{
+	err := GetRepositoryDao(suite.tx).Create(api.RepositoryRequest{
```

Comment:
i like this solution of passing the transaction into the dao, this is a much better way!

Reply:
thanks!